packages/flutter/test/widgets/image_test.dart

```diff
@@ -1956,6 +1957,22 @@ void main() {
       matchesGoldenFile('image_test.missing.2.png'),
     );
   }, skip: kIsWeb); // https://github.com/flutter/flutter/issues/74935 (broken assets not being reported on web)
+
+   testWidgets('Image.file throws a non-implemented error on web', (WidgetTester tester) async {
```

Comment:
Can you also update this test to reflect the style guide? Thanks!

Reply:
Done. PTAL